timemachine/fe/rest/queries.py

```diff
@@ -0,0 +1,29 @@
+from rdkit import Chem
+from rdkit.Chem import rdMolDescriptors
+
+from .bond import CanonicalBond, mkbond
+
+
+def get_rotatable_bonds(mol: Chem.rdchem.Mol) -> set[CanonicalBond]:
```

Comment:
Yes, this is a bit ad-hoc; this definition is actually mostly duplicated from https://github.com/proteneer/timemachine/blob/f4721d90f2475c77a0654a47fbc36283a26df1e7/timemachine/md/enhanced.py#L35, mainly because importing the latter definition resulted in a circular import, but also because I imagined that the two definitions might diverge in the future.

I agree that moving this to a common utility module seems like a sensible way to organize in the future, but am unsure of the right place (e.g. seems a bit weird to import definition from the `fe` submodule into `md`, but maybe this is fine). Thoughts?

Reply:
[organization] Makes sense -- inclined to keep the new file as-is! There are at least 2 other definitions that will probably go here (`get_aliphatic_ring_bonds`, maybe `get_angles_to_soften`), and these might diverge from related non-REST uses.

[flexibility]

>I'd think I'd lean toward waiting to add the strict pattern (or some variant) until we've demonstrated a case where the current definition is too permissive. Thoughts?

Makes sense -- and I don't have a clear adjustment to suggest now. (If there were some known classes of possibly-rotatable bonds that we're missing, or some classes of shouldn't-soften torsions that we're currently hitting, maybe we would want to add toggles for these. I don't have a clear idea of such classes currently.) If this becomes clearer in the future, we can just change the definition of `identify_rotatable_bonds`.